src/Adapter/MSTestAdapter.PlatformServices/Services/ReflectionOperations.cs

```diff
@@ -60,6 +72,6 @@ public object[] GetCustomAttributes(Assembly assembly, Type type) =>
 #if NETFRAMEWORK
         ReflectionUtility.GetCustomAttributes(assembly, type).ToArray();
 #else
-        assembly.GetCustomAttributes(type).ToArray<object>();
```

Comment:
Isn't this line changing behavior by setting inherit to true?

Reply:
That is the default already

https://github.com/dotnet/runtime/blob/2d81cbb28a8feb1af5b790664b511b264d032877/src/coreclr/System.Private.CoreLib/src/System/Attribute.CoreCLR.cs#L697-L701

Reply:
And actually, for assemblies specifically, this parameter has no meaning and is unused.

https://github.com/dotnet/runtime/blob/2d81cbb28a8feb1af5b790664b511b264d032877/src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs#L332-L340